protocol/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/keybase/protocol",
   "dependencies": {
-    "avdl-compiler": "1.4.8",
+    "avdl-compiler": "https://github.com/keybase/node-avdl-compiler.git#xgess/picnic-1133-allow-serializing-nulls",
```

Comment:
will change this once https://github.com/keybase/node-avdl-compiler/pull/52 is merged